health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/utils/downloadExcel.js

```
@@ -0,0 +1,52 @@
+ * Eg. ->
+ *  const handleDownload = (id, name) => {
+ *      downloadExcelWithCustomName({fileStoreId: id, customName: name});
+ *  }
+ *
+ */
+
+export const downloadExcelWithCustomName = ({ fileStoreId = null, customName = null }) => {
+  const downloadExcel = (blob, fileName) => {
+    const link = document.createElement("a");
+    link.href = URL.createObjectURL(blob);
+    link.download = fileName + ".xlsx";
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)